synapse/storage/databases/main/receipts.py

```diff
@@ -165,6 +196,17 @@ async def get_receipts_for_user(
     async def get_receipts_for_user_with_orderings(
         self, user_id: str, receipt_type: str
     ) -> JsonDict:
+        """
+        Fetch receipts in all rooms for a user.
```

Comment:
Nit: it's presumably not all rooms the server can see.

```suggestion
Fetch receipts for all rooms that the given user is joined to.
```